### PR TITLE
Spring Boot 3.x Support

### PR DESCRIPTION
**Pull request by mayur-solace**

### CODE REVIEW

**Review by Nephery — CHANGES_REQUESTED** [review]

File: solace-spring-cloud-parent/pom.xml [hunk 1]
```diff
@@ -20,11 +20,11 @@
...
 
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
 
-        <solace.jcsmp.version>10.19.0</solace.jcsmp.version>
-        <solace.jms.version>10.19.0</solace.jms.version>
+        <solace.jcsmp.version>10.19.1-jakarta_jms.43138</solace.jcsmp.version>
+        <solace.jms.version>10.19.1-jakarta_jms.43138</solace.jms.version>
```
[the remark this answers is not in the document]
Comment (mayur-solace) [reply solace-spring-cloud-parent/pom.xml:27]: release version is not out yet.

File: solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceConsumerProperties.java [hunk 2]
```diff
@@ -17,7 +16,6 @@ public class SolaceConsumerProperties extends SolaceCommonProperties {
 	 * <p>The maximum number of messages per batch.</p>
 	 * <p>Only applicable when {@code batchMode} is {@code true}.</p>
 	 */
-	@Min(1)
```
[the remark this answers is not in the document]
Comment (Nephery) [reply solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceConsumerProperties.java:20]: Looks like there's jakarta equivalents for validation annotations (e.g. `jakarta.validation.constraints.Min`).
Please add the validation annotations back in.

[the remark this answers is not in the document]
Comment (mayur-solace) [reply solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceConsumerProperties.java:20]: Initially plan was to remove both @Min and @PostConstruct. There is any easy way to replace @PostConstruct. So I haven't removed @PostConstruct. I will revert and add @Min back as well. Thanks.

**Review by Nephery — APPROVED** [review]
JCSMP/JMS versions will be updated to release version whenever they come out.

**Pull request merged**